src/lhttpc_lib.erl

```diff
@@ -239,12 +239,26 @@ split_credentials(CredsHostPortPath) ->
             case string:tokens(Creds, ":") of
                 [User] ->
                     % RFC1738 says ":password" is optional
-                    {http_uri:decode(User), "", HostPortPath};
+                    {http_uri_decode(User), "", HostPortPath};
                 [User, Passwd] ->
-                    {http_uri:decode(User), http_uri:decode(Passwd), HostPortPath}
+                    {http_uri_decode(User), http_uri_decode(Passwd), HostPortPath}
             end
     end.
 
+-ifdef(OTP_RELEASE).
+-if(?OTP_RELEASE >= 23).
+http_uri_decode(User0) ->
```

Comment:
I would prefer we name it something else other than `http_uri:decode/1`, since it doesn't actually decode an URI, but rather it decomposes/dissects/parses a query string.

Reply:
What name do you propose? I'll use it as you mention it. I want to waste no more time on this PR.

Reply:
`parse_query_string`?

Reply:
Cool, thx, bye.